src/objects/Attribute__c.object

```diff
@@ -0,0 +1,253 @@
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Expired__c</fullName>
+        <description>Indicates whether the Attribute has expired. Checked automatically when the Attribute end date is before the current date.</description>
+        <externalId>false</externalId>
+        <formula>TODAY() &gt;  End_Date__c</formula>
+        <formulaTreatBlanksAs>BlankAsZero</formulaTreatBlanksAs>
+        <inlineHelpText>Indicates whether the Attribute has expired. Checked automatically when the Attribute end date is before the current date.</inlineHelpText>
+        <label>Expired</label>
+        <trackTrending>false</trackTrending>
+        <type>Checkbox</type>
+    </fields>
+    <fields>
+        <fullName>Start_Date__c</fullName>
+        <description>The date when the credential or Attribute is issued.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The date when the credential or Attribute is issued.</inlineHelpText>
+        <label>Start Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <label>Attribute</label>
+    <listViews>
+        <fullName>All</fullName>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+    </listViews>
+    <nameField>
+        <displayFormat>AT - {0000}</displayFormat>
+        <label>Attribute Name</label>
+        <type>AutoNumber</type>
+    </nameField>
+    <pluralLabel>Attributes</pluralLabel>
+    <recordTypes>
+        <fullName>Credential</fullName>
+        <active>true</active>
+        <description>Allows educational institutions to track licenses, certifications, or endorsements earned by staff or students.</description>
+        <label>Credential</label>
+        <picklistValues>
+            <picklist>Attribute_Type__c</picklist>
+            <values>
+                <fullName>Certification</fullName>
+                <default>false</default>
+            </values>
+            <values>
+                <fullName>Endorsement</fullName>
+                <default>false</default>
+            </values>
+            <values>
+                <fullName>Licensure</fullName>
+                <default>false</default>
+            </values>
+            <values>
+                <fullName>Other</fullName>
+                <default>false</default>
+            </values>
+        </picklistValues>
+    </recordTypes>
+    <recordTypes>
+        <fullName>Student_Characteristic</fullName>
+        <active>true</active>
+        <description>Allows educational institutions to track important student characteristics that might impact their education.</description>
+        <label>Student Characteristic</label>
+        <picklistValues>
+            <picklist>Attribute_Type__c</picklist>
+            <values>
+                <fullName>Accommodations</fullName>
+                <default>false</default>
+            </values>
+            <values>
```

Comment:
@xinnanlisf I don't know that "Certification" should be part of this RecordType (thought a customer could always remove it. Same with Endorsement and Licensure